### DISCUSSION TITLE
Is there a way to expose a Calendar view as a iCal file?

### DISCUSSION THREAD

**Discussion by googollee** [nocodb/nocodb#8537] [Q&A]
I'd like to show a Calendar view in my calendar app. Is there a way to expose a Calendar view as a iCal file? Then I can import this iCal file in other apps.

**Comment by rallisf1** [comment]
I think a Google Calendar integration is better.

iCal doesn't sync, you'll need to import it every day and it can get pretty big depending on your data.

CalDAV would be an alternative but it's a lot harder to integrate.
